php_fpm/tests/test_e2e.py

```diff
@@ -0,0 +1,66 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.base.constants import ServiceCheck
+
+pytestmark = pytest.mark.e2e
+
+
+def test_bad_ping_reply(dd_agent_check, instance, ping_url_tag):
```

Comment:
I dont think we need to replicate all integration tests in e2e, just the happy path. Unless we're exercising different part of the agent